gridkit/_statistical_functions.py

```diff
@@ -30,11 +30,11 @@ def _empty_combined_grid(grids, value=numpy.nan):
         )
 
     combined_bounds = total_bounds(grids)
-    _, shape = grids[0].cells_in_bounds(
+    ids = grids[0].cells_in_bounds(
         combined_bounds
     )  # TODO: split ids and shape outputs into different methods
     combined_data = numpy.full(  # data shape is y,x
-        shape,
+        ids.shape,
```

Comment:
This shape can actually be 1D, which I think could break this logic.
Test if this is the case and if it is, we need to get the shape through `return_cell_count=True`